samples/app-link-unfurling-in-share-to-teams/csharp/assets/sample.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-link-unfurling-in-share-to-teams-csharp",
+    "source": "officeDev",
+    "title": "App link unfurling in share to teams",
+    "shortDescription": "Demonstrating feature of link unfurling in share to teams.",
```

Comment:
Demonstrating the feature of link unfurling in share to teams.

Reply:
changed it